lumicks/pylake/kymotracker/detail/trace_line_2d.py

```diff
@@ -180,14 +182,23 @@ def _traverse_line_direction(
     return nodes
 
 
+@dataclass(frozen=True)
 class KymoLineData:
-    __slots__ = ["time_idx", "coordinate_idx"]
+    """Data class which holds the raw data for a Kymograph line.
+
+    Parameters
```

Comment:
Should these be attributes? AFAIK in NumpyDoc, `Parameters` refer to things passed to functions and `Attributes` refer to members.